code/src/components/Like.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react"
+
+import { LIKES_URL } from "../utils/urls"
+
+const Like = ({ thought, hearts, setHearts, likeCounter, setLikeCounter }) => {
+    const [isLiked, setIsLiked] = useState(false)
+
+    const onHeartChange = () => {
+
+        setIsLiked(true)
+        setLikeCounter(likeCounter + 1)
+
```

Comment:
Very good naming in the whole project. Makes it easy to follow and understand the code.